docs/technical_documents/open_space_decider.md

```diff
@@ -0,0 +1,175 @@
+# OPEN SPACE DECIDER
+
+# Introduction
+
+Apollo planning is a scenario based planning method, each driving use case is treated as a different driving scenario.
```

Comment:
Maybe: Apollo planning is scenario-based, where each driving case is treated as a different driving scenario.